tests/integration/dhcp_test.py

```diff
@@ -219,3 +219,79 @@ def _clean_up():
         os.unlink(DNSMASQ_CONF_PATH)
     except FileNotFoundError:
         pass
+
+
+def test_slave_ipaddr_learned_via_dhcp_added_as_static_to_linux_bridge(dhcp_env
+                                                                       ):
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'dhcpcli',
+                'type': 'ethernet',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': True
+                },
+            }
+        ]
+    }
+
+    netapplier.apply(desired_state)
+
+    current_state = statelib.show_only(('dhcpcli',))
+    client_current_state = current_state[INTERFACES][0]
+    dhcpcli_ip = client_current_state['ipv4']['address']
+
+    bridge_desired_state = {
+        INTERFACES: [
+            {
+                'name': 'linux-br0',
+                'type': 'linux-bridge',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': False,
+                    'address': dhcpcli_ip
+                },
+                'bridge': {
+                    'options': {
+                        'group-forward-mask': 0,
+                        'mac-ageing-time': 300,
+                        'multicast-snooping': True,
+                        'stp':
+                            {'enabled': True,
+                             'forward-delay': 15,
+                             'hello-time': 2,
+                             'max-age': 20,
+                             'priority': 32768
+                             }
+                     },
+                    'port': [
+                        {
+                         'name': 'dhcpcli',
+                         'stp-hairpin-mode': False,
```

Comment:
IMHO it would be great to keep the name only here.

Reply:
Current change is causing the bug - port verification is failed in assert_ifaces_state func